hack/configure-self-signed-cluster.md

```diff
@@ -27,7 +21,7 @@ Next, run the install script as usual
 Once the fulcio-system ns exists, create the ingress-cert secret in the fulcio-system namespace
 
 ```bash
-oc create secret generic -n fulcio-system clustercert --from-file=/path/to/clustercert/tls.crt
+oc create secret generic -n fulcio-system clustercert --from-file=/var/run/clustercert/tls.crt
```

Comment:
this --from-file can be anywhere, doesn't have to match the path in the container (so can be non-root location), here you pass the local path to the file, wherever it is

Reply:
Thanks! That's super interesting, so how does the container know where to look for this certificate?

Reply:
answered in slack but I'll also put here for others :) 

`oc create secret generic -n fulcio-system clustercert --from-file=/path/on/local/clustercert/tls.crt` creates the secret.
then this is how/where the file from the secret is placed in the fulcio-server container (the patch file):
```
spec:
  template:
    spec:
      containers:
         env:
        - name: SSL_CERT_DIR
          value: /var/run/clustercert     
      volumeMounts:
        - mountPath: /var/run/clustercert
          name: clustercert
          readOnly: true
      volumes:
      - name: clustercert
        secret:
          defaultMode: 420
          items:
          - key: tls.crt
            path: tls.crt
          secretName: clustercert
```
the secret is in the fulcio-system ns - and it contains a `key: tls.crt` with `value: contents-of-tls.crt` and whatever you put as the `mountPath`, that dir will be created in the container, so there will be `/var/run/clustercert/tls.crt` will be in the fulcio container